RubrikSecurityCloud/RubrikSecurityCloud.PowerShell/Cmdlets/Invoke-Rsc.cs

```diff
@@ -122,10 +122,16 @@ internal void ProcessRecord_Query()
                 {
                     throw new ArgumentException("No GQL query provided");
                 }
-                var _asPathString = Path.GetFullPath(sQuery);
-                if (File.Exists(_asPathString))
-                {
-                    sQuery = File.ReadAllText(_asPathString);
+                
+                // if the content of sQuery is an existing file, read it
+                try {
+                    var _asPathString = Path.GetFullPath(sQuery);
+                    if (File.Exists(_asPathString))
+                    {
+                        sQuery = File.ReadAllText(_asPathString);
+                    }
+                } catch (Exception) {
+                    // ignore
```

Comment:
the wanted behavior is: if the passed string is a filename, read from it, otherwise use it as is , so we don't actually want to handle this exception.
We didn't need the try catch in poewrshell 7, but we do for 5.1